lib/juttle-subprocess.js

```diff
@@ -84,7 +81,7 @@ process.on('message', function(msg) {
 
         var compile_options = {
             stage: 'eval',
-            fg_processors: [implicit_views(config.implicit_view), optimize, views_sourceinfo],
+            implicit_view: config.implicit_view,
```

Comment:
Not sure I follow its always been at the top of the config object as it was before I changed this line ? This is the case here for the config and also for the CLI.